VocaDbWeb/Views/Shared/Partials/_LayoutScripts.cshtml

```diff
@@ -20,7 +20,7 @@
 
     vdb = vdb || {};
     vdb.values = vdb.values || {};
-    vdb.values.hostAddress = '@UrlMapper.HostAddress';

```

Comment:
Did you review all places where this is used, so it can be relative instead of absolute?

Reply:
Actually, now that I'm checking context, baseAddress already contains the relative path (RootPath), right? So if absolute path is not needed, it's best to just remove this hostAddress variable and migrate remaining usages to baseAddress

Reply:
Removed `hostAddress` variable from GlobalValues.ts.